flow/embedded_views.h

```diff
@@ -6,11 +6,15 @@
 #define FLUTTER_FLOW_EMBEDDED_VIEWS_H_
 
 #include "flutter/fml/memory/ref_counted.h"
+#include "third_party/skia/include/core/SkPoint.h"
+#include "third_party/skia/include/core/SkSize.h"
 
 namespace flow {
 
 class EmbeddedViewParams {
```

Comment:
Lets add an equality check for the embedded view params and ask the view embedder to composite the same only if the equality check from the last frame changed.

It is fine to do this in a later patch with a TODO added here.

Reply:
Added a TODO in FlutterPlatformViews.mm (to make sure we don't do anything if the params didn't change)